inception-ui-core/src/main/java/de/tudarmstadt/ukp/inception/ui/core/dashboard/project/ProjectDashboardPage.java

```diff
@@ -51,7 +56,7 @@
 /**
  * Project dashboard page
  */
-@MountPath(value = "/project/${" + PAGE_PARAM_PROJECT_ID + "}")
+@MountPath(value = "/project/${" + PAGE_PARAM_PROJECT_ID + "}/#{" + PAGE_PARAM_INVITE_ID + "}")
```

Comment:
Hm, why use a HTML anchor tag (`#`) here instead of a query parameter (`?`)?

Reply:
The `#` is wicket's way to mark an optional part of the url, an example invite url would be something like `https://blabla.com/inception/projects/2/NMshwsuwhdw_`